addons/mail/static/src/js/composers/mention_manager.js

```diff
@@ -315,7 +313,9 @@ var MentionManager = Widget.extend({
         }
         if (suggestions.length) {
             this.$el.html(QWeb.render(this._activeListener.suggestionTemplate, {
-                suggestions: suggestions,
+                suggestions: _.map(suggestions, function (suggestion) {
+                   return {'id': suggestion['id'], 'public': suggestion['public'], 'name': _.unescape(suggestion['name'])};
+                })
```

Comment:
unescaping only for UI rendering (avoid having this : "Project &amp; others" but having this instead : "Project & others")

Reply:
```js
// mention manager stores escaped suggestion names
return _.extend({}, suggestion, { name: _.unescape(suggestion.name) });
```